hnn_core/viz.py

```diff
@@ -182,7 +185,13 @@ def plot_laminar_lfp(times, data, contact_labels, tmin=None, tmax=None,
             col = color
         ax.plot(plot_times, plot_data + trace_offsets[contact_no],
                 label=f'C{contact_no}', color=col)
-
+        # To be removed after deprecation cycle
+        if tmin is not None or tmax is not None:
+            ax.set_xlim(left=tmin, right=tmax)
+            warnings.warn('tmin and tmax are deprecated and will be removed in future releases of hnn-core.'
+                          'By default, dipoles and laminar LFPs are now plotted from 0 to tstop.', DeprecationWarning)
```

Comment:
```suggestion
                          'Please use matplotlib directly to set tmin and tmax', DeprecationWarning)
```

nice to tell the user what they should do instead